### ISSUE TITLE
quoting using @username

### ISSUE THREAD

**Issue by ghost** [wetfish/online#37]
quote function using @username for the most recent post

logic for posts further than that?

**Comment by itsrachelfish** [comment]
huh that's a cool idea, I think it should be like how github autocompletes issues / pull requests when you type "#". A small autocomplete list of that user's recent posts in the thread should appear. For example, see the attached image.

![screenshot from 2016-12-03 23 24 42](https://cloud.githubusercontent.com/assets/1670549/20871899/57696436-ba57-11e6-8d2b-7a4238d3c913.png)

**Comment by ghost** [comment]
It's probably been a long time since I've been on a forum but quoting posts is troublesome on the phone and just a hassle on the computer.
If we were to implement @username quoting wouldn't we need to define the post, in that thread, the user is trying to quote?

Maybe a pop up drop down going from the most recent from the users post they are creating. Up to 5 lines by default? Settings could include a range you could select.

**Comment by ghost** [comment]
I'm noticing loss of context involving avatars when quoting as well.

![2016-12-08 14_06_48-act like a 12 year old - wetfish online](https://cloud.githubusercontent.com/assets/17911663/21025859/b6c0cf5a-bd4f-11e6-86bf-bf2db8bd426c.png)

There might be another Issue that brings up quote expansion that I'm forgetting about atm. Maybe include a slightly opaque avatar in the background of the quote and when hovered over it expands?

**Comment by itsrachelfish** [comment]
Yeah! Actually it'd be really cool if a quote would have a copy of what that user's avatar looked like at the time.
